zingoconfig/src/lib.rs

```diff
@@ -28,12 +28,12 @@ pub const MAX_REORG: usize = 100;
 pub const WALLET_NAME: &str = "zingo-wallet.dat";
 pub const LOGFILE_NAME: &str = "zingo-wallet.debug.log";
 pub const REORG_BUFFER_OFFSET: u32 = 0;
-pub const GAP_RULE_UNUSED_ADDRESSES: usize = if cfg!(any(target_os = "ios", target_os = "android"))
-{
-    0
-} else {
-    5
-};
+
+#[cfg(any(target_os = "ios", target_os = "android"))]
+pub const GAP_RULE_UNUSED_ADDRESSES: usize = 0;
+
+#[cfg(not(any(target_os = "ios", target_os = "android")))]
+pub const GAP_RULE_UNUSED_ADDRESSES: usize = 5;
```

Comment:
Is this the code you said was dead code?

Reply:
not in zingolib but pub so will leave for now